src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/RangerCombatAI.java

```diff
@@ -356,5 +356,6 @@ protected void onTargetDied(final LivingEntity entity)
     {
         parentAI.incrementActionsDoneAndDecSaturation();
         user.getCitizenExperienceHandler().addExperience(EXP_PER_MOB_DEATH);
+        user.getCitizenColonyHandler().getColony().getStatisticsManager().increment(MOBS_KILLED);
```

Comment:
The nether miner should probably have one of these too.  Unless you want to count that separately.  Lots of possibly interesting expedition counters.

Reply:
Separately probably